CharlieBackend.Business/Services/LessonService.cs

```diff
@@ -52,8 +53,9 @@ public async Task<LessonDto> CreateLessonAsync(CreateLessonDto lessonDto)
                 }
 
                 await _unitOfWork.CommitAsync();
+                var createdLesson = _mapper.Map<LessonDto>(createdLessonEntity);
 
-                return _mapper.Map<LessonDto>(createdLessonEntity);
+                return Result<LessonDto>.GetSuccess(createdLesson);
```

Comment:
Get back in return object mapping

Reply:
We should keep code more simple, unnecessary properties make it harder